### ISSUE TITLE
Django 3.2

### ISSUE THREAD

**Issue by torgeirl** [devilry/trix2#102]

**Comment by Levijatan** [comment]
I have updated trix to django 3.2.16, and bumped all packages to the newest possible version while supporting python 3.6 -> 3.11. Made a 3.0.0b11 release of the changes.

#107 was fixed as a side effect of upgrading markedown.

I can't update to django 4 before cradmin-legacy gets support for it or we remove it from trix

**Comment by torgeirl** [comment]
I think Django should be latest patch (`3.2.*`) instead of `3.2.16`. And at least Python 3.6 support can now be dropped (End-of-Life in December 2021).

**Comment by torgeirl** [comment]
The `Markdown` deprecation (#107) is now fixed, but there is suddenly a manual conformation step with the federated login (#117)

Migrations when updating b10 => b11:
```
$ oc logs job.batch/trix-dev-migration
Begin migration ...
Operations to perform:
  Apply all migrations: account, admin, auth, contenttypes, sessions, sites, socialaccount, trix_core
Running migrations:
  Applying admin.0003_logentry_add_action_flag_choices... OK
  Applying auth.0009_alter_user_last_name_max_length... OK
  Applying auth.0010_alter_group_name_max_length... OK
  Applying auth.0011_update_proxy_permissions... OK
  Applying auth.0012_alter_user_first_name_max_length... OK
  Applying trix_core.0007_auto_20221130_1209... OK
Migration completed
```

**Comment by Levijatan** [comment]
> I think Django should be latest patch (`3.2.*`) instead of `3.2.16`. And at least Python 3.6 support can now be dropped (End-of-Life in December 2021).

3.2.16 is the latest version but yes could change it to 3.2.*. or >=3.2.16, <4 to specify it a bit more.

**Comment by Levijatan** [comment]
Created a new release b12 that contains some small fixes stemming from the django upgrade, there where some devtools that needed some strange fixes. And also contains fixes for the other issues in the milstone.

**Comment by torgeirl** [comment]
@Levijatan: I believe this issue and the 3.0 milestone as a whole now is completed. :+1:

**Comment by Levijatan** [comment]
The 3.0 Milestone is now merged into master and version 3.0.0 is released.
